platformio.ini

```diff
@@ -1362,6 +1362,8 @@ lib_ignore    = NativeEthernet
 upload_speed  = 115200
 #upload_port   = marlinesp.local
 #board_build.flash_mode = qio
+# Please delete '#' of following line if you use FYSETC E4 board
+#board_build.partitions = default_16MB.csv
```

Comment:
You can easily extend the environment to add this detail. It's up to thinkyhead if he'd like commented lines like this or environments. It would be something like this:

```
[env:fysetc_e4]
platform               = espressif32@1.11.2
extends                = env:esp32
board_build.partitions = default_16MB.csv

Reply:
Yes , this is good choice.

Reply:
Actually, you can even use:
`platform               = ${env:esp32.platform}`

This will ensure it always matches the base environment.

(Platform IO does not require `platform` if you are extending another environment, but the VS Code plugin does not work properly without it.)